### PR TITLE
Support config the default body encoding, Ignore sampling trace when the `MinDuration` is not set

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]

File: configs/rover_configs.yaml [hunk 1]
```diff
@@ -130,4 +130,12 @@ profiling:
...
\ No newline at end of file
+        queue_size: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_QUEUE_SIZE:5000}
+        # The profiling config of the protocols
+        sampling:
+          # The HTTP/1.x and HTTP/2.x profiling config
+          http:
+            # The default body encoding when sampling the request
+            default_request_encoding: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_SAMPLING_HTTP_DEFAULT_REQUEST_ENCODING:UTF-8}
+            # The default body encoding when sampling the response
+            default_response_encoding: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_SAMPLING_HTTP_DEFAULT_RESPONSE_ENCODING:UTF-8}
```
Comment (wu-sheng) [inline configs/rover_configs.yaml:141]: What happens if not set?

Comment (mrproliu) [reply configs/rover_configs.yaml:141]: For now, I'm not handling this situation, should I add a default value?

Comment (wu-sheng) [reply configs/rover_configs.yaml:141]: If the rover exits, rather than running unexpected, it is fine.

Comment (mrproliu) [reply configs/rover_configs.yaml:141]: Sure, I could add a check when the rover start.

Comment (wu-sheng) [reply configs/rover_configs.yaml:141]: You could fall back to utf8 or check and fail the booting. Your call.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**